src/java.base/share/classes/java/net/Socket.java

```diff
@@ -1095,6 +1098,20 @@ public int read() throws IOException {
         }
         @Override
         public int read(byte[] b, int off, int len) throws IOException {
+            if (! SocketReadEvent.enabled()) {
```

Comment:
Drop the space in "! SocketReadEvent" as it is inconsistent with the existing code.